packages/blocks/src/api/raw-handling/head-remover/index.js

```diff
@@ -0,0 +1,10 @@
+export default function( node ) {
+	if (
+		node.nodeName !== 'META' &&
+		node.nodeName !== 'STYLE'
```

Comment:
Interesting that we stopped stripping these. Maybe this happened with the paste rewrite, but that was ages ago... If we want to continue with this fix, maybe it's not a bad idea to think about all HTML tags that should be stripped entirely (as opposed to unwrapped), like `script` etc.

Reply:
Yeah, `script` would be another one. I can’t see a use for just unwrapping them.

Reply:
I'd say `script`, `style`, `noscript` and `template`. `meta` is fine I think, it cannot have any child nodes. I wouldn't expect it in pasted HTML, but maybe it's prudent to strip `head` as well.